### PR TITLE
feat: cleanup ui

### PR DESCRIPTION
**Pull request by gregfromstl**
- Reduced border radius on buttons
- Reduced connected button size
- Removed layout shift between connected and unconnected states
- Remove Rainbow from default wallets (replaced with Rabby)

https://github.com/user-attachments/assets/334f09d5-46d1-40e3-8b3a-5937771f277c

---

## PR-Codex overview
This PR focuses on UI cosmetic improvements in the `thirdweb` package.

### Detailed summary
- Added a new wallet type `io.rabby`
- Updated padding and border radius in various UI components for consistency
- Adjusted button sizes and styles for better visual appeal

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/thirdweb/src/react/web/wallets/defaultWallets.ts [hunk 1]
```diff
@@ -19,11 +14,11 @@ export function getDefaultWallets(options?: {
   return [
     createWallet("inApp"),
     createWallet(METAMASK),
+    createWallet("io.rabby"),
```
Comment (joaquim-verges) [inline packages/thirdweb/src/react/web/wallets/defaultWallets.ts:17]: any reason for this? i like rainbow :D

Comment (gregfromstl) [reply packages/thirdweb/src/react/web/wallets/defaultWallets.ts:17]: Mike wasn't a fan because of RainbowKit

Comment (joaquim-verges) [reply packages/thirdweb/src/react/web/wallets/defaultWallets.ts:17]: i think its more popular than rabby though, but lets see if ppl complain i guess

**Review by joaquim-verges — APPROVED** [review]

File: packages/thirdweb/src/react/web/wallets/defaultWallets.ts (hunk 1 above)
Comment (joaquim-verges) [reply packages/thirdweb/src/react/web/wallets/defaultWallets.ts:17]: wait but rabby above coinbase?

**Pull request merged**